xml/deployment_additional.xml

```diff
@@ -66,31 +66,23 @@
   </procedure>
  </sect1>
  <sect1 xml:id="installing.dns">
-  <title>Installing DNS Server</title>
+  <title>Installing Kubernetes DNS</title>
 
   <para>
-   Even though internal DNS server is essential for proper function of
-   &admin_node; and &kmaster;, it is not deployed automatically and you have to
-   deploy it manually. To install the DNS server, run the following command:
+   Kubernetes DNS is required if you intend to use DNS for service discovery
+   in your Kubernetes cluster. Kubernetes DNS is not deployed automatically
+   upon cluster provisioning and needs to be manually installed. In order to
+   install Kubernetes DNS, run the following command:
   </para>
 
-<screen>kubectl apply -f https://raw.githubusercontent.com/SUSE/caasp-services/master/contrib/addons/kubedns/dns.yaml</screen>
+  <screen>kubectl apply -f https://raw.githubusercontent.com/SUSE/caasp-services/master/contrib/addons/kubedns/dns.yaml</screen>
 
   <para>
-   After the command finishes, SSH to each
-   <emphasis role="bold">&kworker;</emphasis> and to the
-   <literal>KUBELET_ARGS</literal> section of the
-   <filename>/etc/kubernetes/kubelet</filename> file add the following entries
-   according to your system:
+   After Kubernetes DNS has been installed, you will be able to resolve pods
+   and services using their names from within the kubernetes cluster. To
+   construct the fully qualified domain name for a pod or service, use
+   '.cluster.local' as the cluster domain suffix.
   </para>
 
-<screen>--cluster-dns=172.21.0.2 \
---cluster-domain=cluster.local \</screen>
-
-  <para>
-   On each &kworker; run the following:
-  </para>
-
-<screen>rckubelet restart</screen>
```

Comment:
Yes, these steps are no longer required. Once Kubernetes DNS is installed, it will be available to the entire cluster (through resolv.conf entries that are already being set). No further action is needed.